js/views/modals/purchase/Purchase.js

```diff
@@ -250,18 +251,20 @@ export default class extends BaseModal {
   disableModerators() {
     this.getCachedEl('#purchaseModerated').prop('checked', false);
     this.order.moderated = false;
+    this.moderators.deselectOthers();
     this.order.set('moderator', '');
   }
 
-  toggleModeration(bool) {
+  moderationOn(bool) {
```

Comment:
This was renamed to make the it more clear what true or false for the passed in value will do.